config/authorino/rbac/role.yaml

```diff
@@ -0,0 +1,45 @@
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  name: manager-role
```

Comment:
I think this role name will collide with the operator's `manager-role`.

Reply:
kustomizer will add an `authorino` prefix name to the resource. https://github.com/Kuadrant/authorino-operator/blob/foundation2/config/authorino/kustomization.yaml#L12